manifests/init.pp

```diff
@@ -68,6 +70,7 @@
   Boolean $use_ssh_fork = $katello_devel::params::use_ssh_fork,
   Optional[String] $fork_remote_name = $katello_devel::params::fork_remote_name,
   String $upstream_remote_name = $katello_devel::params::upstream_remote_name,
+  Integer[0, 1000] $qpid_wcache_page_size = $::katello::params::qpid_wcache_page_size,
```

Comment:
Could this be `Integer[0, 1000] $qpid_wcache_page_size = $::katello_devel::params::qpid_wcache_page_size` since the katello_devel params are inherited from katello?